load_test.py

```diff
@@ -201,10 +200,14 @@ def main(args):
 
         utils.write_output(config, results_list)
 
+        while not dataset_q.empty():
+            logging.warning("Removing more elements from dataset_q after gathering results!")
+            dataset_q.get()
+
```

Comment:
Should we add `dataset_q.close()` here after we removed all the elements?

Reply:
@dagrayvid I understand that this hunk is emptying the queue, and nothing else, right? (at the functional level)

in this case, you may want to move the log message out of the loop. Something like this:
```suggestion
        if not dataset_q.empty():
            logging.warning(f"Removing {len(dataset_q)} elements from dataset_q after gathering results!")
            while not dataset_q.empty():
                dataset_q.get()
```